lnd.go

```diff
@@ -170,14 +170,18 @@ type ListenerWithSignal struct {
 
 	// Ready will be closed by the server listening on Listener.
 	Ready chan struct{}
+
+	// MacChan is an optional way to pass the admin macaroon to the program
+	// that started lnd.
```

Comment:
We should probably add a sentence here mentioning that the channel should be buffered to avoid lnd being blocked on sending to the channel.